nautobot/ipam/tables.py

```diff
@@ -32,7 +32,7 @@
 
 UTILIZATION_GRAPH = """
 {% load helpers %}
-{% if record.present_in_database %}{% utilization_graph record.get_utilization %}{% else %}&mdash;{% endif %}
+{% if record.pk %}{% utilization_graph record.get_utilization.0 record.get_utilization.1 %}{% else %}&mdash;{% endif %}
```

Comment:
The change from `present_in_database` to `pk` is incorrect, please revert it.

Same suggestion as above about having `get_utilization` return a dict or namedtuple instead of a raw tuple so we don't have to refer to `0` and `1` as magic indices.